src/rest.cpp

```diff
@@ -295,12 +306,15 @@ static bool rest_mempool_info(HTTPRequest* req, const std::string& strURIPart)
 {
     if (!CheckWarmup(req))
         return false;
+    const CTxMemPool* mempool_ = EnsureMemPool(req);
+    if (!mempool_) return false;
```

Comment:
Or it could just throw an exception leaving the request handlers free of handling these error case. Only change here would be like
```diff
-UniValue mempoolInfoObject = MempoolInfoToJSON(::mempool);
+UniValue mempoolInfoObject = MempoolInfoToJSON(GetMemPoolOrThrow());
```